src/sdk/pynni/nni/compression/torch/utils/mask_conflict.py

```diff
@@ -122,7 +122,8 @@ def fix_mask(self):
                 w_shape = module.weight.data.size()
                 w_mask = torch.ones(w_shape).to(device)
                 b_mask = None
-                if hasattr(module, 'bias'):
+                if hasattr(module, 'bias') and module.bias is not None:
```

Comment:
why the config_list in issue 2709 can trigger this problem?

Reply:
This bug will be triggered only when some input convolutional layers of the cat operation are pruned, and the others are not. In the unit test, we will randomly assign a sparsity for each conv layer, so all the conv layers are pruned in the model.